### DISCUSSION TITLE
Ran update but still on 3.5.5?

### DISCUSSION THREAD

**Discussion by MaryMuse** [mastodon/mastodon#25257] [Q&A] [answered]
![Screenshot 2023-06-02 150703](https://github.com/mastodon/mastodon/assets/36464631/31cba4ba-fc92-49a0-8b25-6de09580d753)
I ran the update instructions from 3.5.5 here --> https://blog.jamie.holdings/2022/11/20/upgrading-mastodon-digitialocean-droplet-to-4-0-2/ (cross-checking with the official docs https://github.com/mastodon/mastodon/releases/tag/v4.0.0) and when I go to my domain, I still see version 3.5.5. Everything seems functional. I can send a message, etc., but I should be on 4.0.2 based on those instructions.

Thoughts? Ideas? Help?

**Comment by renchap** [comment]
This step most probably did not work and displayed an error or something:

`git fetch && git checkout v4.0.2`

Can you retry and paste the output of the commands here?

**Comment by MaryMuse** [comment]
The first time it ran without errors. Now when I run it, I get this:

![image](https://github.com/mastodon/mastodon/assets/36464631/9fcedb55-65ce-4bb7-8fd6-f913a8ace981)

**Reply by renchap** [reply, marked as answer]
What does `git status` says?

You can run `git checkout yarn.lock` to reset this file to the git version.

If the frontend is showing 3.5.5 and you restarted everything, then it means your `git checkout` did not work the first time (or you forgot this step).

**Comment by MaryMuse** [comment]
Fascinating. I ran the git checkout yarn.lock to rest it, and then ran git status. I noticed when I went through everything again, I did get a message after git checkout saying I was updated and then completed the rest of the install steps and it worked this time. THANK YOU!!! (Although I'm sad to get rid of "toot" in favor of publish! 😃 )
